src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/casenotes/CaseNotesReconciliationService.kt

```diff
@@ -177,48 +177,53 @@ class CaseNotesReconciliationService(
 
   private val truncatedToSecondsFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss")
 
-  private fun CaseNote.transformFromDps() = ComparisonCaseNote(
-    id = caseNoteId,
-    text = text.truncateToFixNomisMaxLength(),
-    type = type,
-    subType = subType,
-    occurrenceDateTime = occurrenceDateTime.format(truncatedToSecondsFormatter),
-    creationDateTime = creationDateTime.format(truncatedToSecondsFormatter),
-    dpsUsername = authorUsername,
-    amendments = amendments.map { a ->
+  private fun CaseNote.transformFromDps(): ComparisonCaseNote {
+    val sortedAmendments = amendments.map { a ->
```

Comment:
not sure why we need to sort both lots of amendments, would have thought that they would arrive in the right order...